modules/nw-egress-ips-about.adoc

```diff
@@ -26,16 +26,28 @@ An egress IP address is implemented as an additional IP address on the primary n
 [id="nw-egress-ips-platform-support_{context}"]
 == Platform support
 
-While you can use the egress IP address functionality on any platform that {product-title} supports, the following platform specific considerations apply:
+Support for the egress IP address functionality on various platforms is summarized in the following table:
 
-Cloud platforms::
-Additional IP addresses must be assigned manually to appropriate nodes by an administrator.
+[IMPORTANT]
+====
+The egress IP address implementation is not compatible with Amazon AWS, Azure Cloud, or any other public cloud platform that does not support manipulation of layer 2 network traffic.
+====
+
+[cols="1,1",options="header"]
+|===
+
+| Platform | Supported
 
-{rhosp}::
-Additional configuration by an administrator is necessary to allow {product-title} to automatically assign additional IP addresses to nodes.
+| bare metal | Yes
+| vSphere | Yes
+| {rhosp} | No
+| Public cloud^[1]^ | No
 
-vSphere::
-Additional configuration by an administrator is necessary to allow {product-title} to automatically assign additional IP addresses to nodes.
+|===
+[small]
+--
+1. Public cloud platforms, such as Amazon AWS and Google GCP, do not support automatic assignment of an egress IP address.
```

Comment:
I would rephrase here too, to the following: `do not support automatic assignment of an egress IP address following OpenShift's current assignment procedure.`. For the same reason I mentioned above.